backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -257,45 +260,77 @@ async def upload_boundaries_excel_sheet(
     get_authorized_request_info(request_info)
 
     try:
+        # Create input temporary file
         input_temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=".xlsx")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)